src/Powercord/plugins/pc-spotify/commands/play.js

```diff
@@ -3,7 +3,7 @@ const urlRegex = /\/track\/([A-z0-9]*)/;
 module.exports = {
   command: 'play',
   description: 'Play a Spotify URL',
-  usage: '{c} <URL>',
+  usage: '{c} < URL >',
```

Comment:
Here we can do the same and close up the spacing before and after "URL".
```suggestion
  usage: '{c} <URL>',
```